Clean up code for converting website and utilize similar code path for latex

aeslaughter opened the pull request:
(refs #8329)

aeslaughter committed:
Correct problem with MooseDocs test command

(refs #6699)

aeslaughter committed:
Clean up code for converting website and utilize similar code path for latex creation.

(refs #8329)

moosebuild commented:
Job `Moose Documentation:linux-clang` on 835f5d7 wanted to post the following:

View the site at http://mooseframework.org/docs/PRs/8343/site

aeslaughter committed:
Allow Mathjax location to configurable

This is needed to allow for internal websites to use a local version of MathJax
(refs #8329)

moosebuild commented:
Job `Moose Documentation:linux-clang` on dc4defd wanted to post the following:

View the site at http://mooseframework.org/docs/PRs/8343/site

moosebuild commented:
Job `Moose Documentation:linux-clang` on 65548d0 wanted to post the following:

View the site at http://mooseframework.org/docs/PRs/8343/site

moosebuild commented:
Job `Moose Documentation:linux-clang` on c915cf0 wanted to post the following:

View the site at http://mooseframework.org/docs/PRs/8343/site

brianmoose reviewed (dismissed):
The menus seem to be a bit wonky on this. They require a click and are placed weird.
Slideshow and flow chart also seem to be broken.

aeslaughter commented:
@brianmoose Nice catch, a missing comma in the init.js script caused all of that.

moosebuild commented:
Job `Moose Documentation:linux-clang` on 04cffea wanted to post the following:

View the site at http://mooseframework.org/docs/PRs/8343/site
